### ISSUE TITLE
Code coverage: END and FINALLY considered not covered

### ISSUE THREAD

**Issue by stefandrissen** [Riverside-Software/sonar-openedge#373]
Using OpenEdge 11.7 x64 (Windows)
````
IF TRUE THEN DO:
   /* something */
END.

RETURN "0":u.

FINALLY:
   /* finally */
END FINALLY.
````

The 'END.' and the 'FINALLY:' lines are considered not covered, resulting in a 60% score.

![image](https://user-images.githubusercontent.com/5297464/30317365-3bda65ae-97aa-11e7-82c9-7f32c3aad4d7.png)

**Comment by gquerret** [comment]
Using OpenEdge 11.6 ? Or lower ?

**Comment by stefandrissen** [comment]
11.7

**Comment by gquerret** [comment]
Using 11.7.1:
![image](https://user-images.githubusercontent.com/6498723/30317596-ce42d836-97aa-11e7-9208-dd1a67ca1df0.png)

Until 11.6, there were many problems with DO: END. blocks
The FINALLY keyword is still considered not executed in 11.7. That's however something to be reported to Progress.

**Comment by stefandrissen** [comment]
Just a small heads up that this has ``finally`` (pun intended) been fixed in 12.8.4 - see [OCTA-9112](https://docs.progress.com/bundle/openedge-product-notes/page/Issues-fixed-in-OpenEdge-12.8.4.html).
